ionoscloud/resource_dbaas_pgsql_cluster.go

```diff
@@ -208,6 +214,11 @@ func resourceDbaasPgSqlCluster() *schema.Resource {
 	}
 }
 func checkDBaaSClusterImmutableFields(_ context.Context, diff *schema.ResourceDiff, _ interface{}) error {
+
+	allowReplace := diff.Get("allow_replace").(bool)
```

Comment:
@cciopa-ionos please also add information about `allow_replace` attribute in `dbaas_pgsql_cluster.md` doc. Also, please take a look into `k8s_cluster.md` file, there we have a warning for the `allow_replace` attribute, we need the same warning here for the PGSQL cluster.